plugin/system_copy.vim

```diff
@@ -22,6 +26,12 @@ function! s:system_copy(type, ...) abort
   echohl String | echon 'Copied to system clipboard via: ' . mode | echohl None
 endfunction
 
+function! s:system_paste() abort
+  let command = <SID>PasteCommandForCurrentOS()
+  put =system(command)
```

Comment:
I found this function a bit confusing on first read. I think we can simplify be dropping the `echohl` line below, and inlining the `command` variable. With that, the function is just one line and to me a bit more clear:

``` vim
function! s:system_paste() abort
  put =system(s:PasteCommandForCurrentOS())
endfunction
```